lang/rs/cndprs/fwd.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
Does Rust not have a json-c package?

Reply:
I tried it and it works. But Rust JSONC crate (Hjson) depends on older version (0.8) of serde JSON and won't have some newer features of serde JSON.  Also newer version of serde JSON is stable version (1.0).  If we prefer JSONC file then using Hjson which depends on older version of serde is an option. I'll update PR to use Hjson.